frontend/model/notifications/selectors.js

```diff
@@ -39,17 +39,53 @@ sbp('sbp/selectors/register', {
       timestamp: data.createdDate ? new Date(data.createdDate).getTime() : Date.now(),
       type
     }
+    const rootState = sbp('chelonia/rootState')
```

Comment:
(Commenting not on this line but the file as a whole)

This file seems misplaced to me. It is not concerned with state management (model), but rather performing actions. So it should be somewhere under `controller/`.

Another thing that's unclear to me is where exactly under `controller/` it should be. We should be able to post notifications when there are no open tabs right? So this should probably be moved under `controller/actions/notifications/selectors.js`